src/Management/test/Endpoint.Test/Actuators/CloudFoundry/CloudFoundrySecurityMiddlewareTest.cs

```diff
@@ -365,6 +365,72 @@ public async Task Redacts_HTTP_headers()
         logMessages.Should().Contain("Authorization: *");
     }
 
+    [InlineData("unavailable", HttpStatusCode.ServiceUnavailable, PermissionsProvider.CloudfoundryNotReachableMessage, "true")]
+    [InlineData("unavailable", HttpStatusCode.OK, PermissionsProvider.CloudfoundryNotReachableMessage, "false")]
+    [InlineData("not-found", HttpStatusCode.Unauthorized, PermissionsProvider.InvalidTokenMessage, "true")]
+    [InlineData("not-found", HttpStatusCode.Unauthorized, PermissionsProvider.InvalidTokenMessage, "false")]
+    [InlineData("unauthorized", HttpStatusCode.Unauthorized, PermissionsProvider.InvalidTokenMessage, "true")]
+    [InlineData("unauthorized", HttpStatusCode.Unauthorized, PermissionsProvider.InvalidTokenMessage, "false")]
+    [InlineData("forbidden", HttpStatusCode.Forbidden, PermissionsProvider.AccessDeniedMessage, "true")]
+    [InlineData("forbidden", HttpStatusCode.Forbidden, PermissionsProvider.AccessDeniedMessage, "false")]
+    [InlineData("timeout", HttpStatusCode.ServiceUnavailable, PermissionsProvider.CloudfoundryNotReachableMessage, "true")]
+    [InlineData("timeout", HttpStatusCode.OK, PermissionsProvider.CloudfoundryNotReachableMessage, "false")]
+    [InlineData("exception", HttpStatusCode.InternalServerError, @"Exception of type \u0027System.Net.Http.HttpRequestException\u0027 was thrown.", "true")]
+    [InlineData("exception", HttpStatusCode.OK, @"Exception of type \u0027System.Net.Http.HttpRequestException\u0027 was thrown.", "false")]
+    [InlineData("no_sensitive_data", HttpStatusCode.OK, null, "true")]
+    [InlineData("success", HttpStatusCode.OK, null, "true")]
+    [Theory]
+    public async Task CloudControllerScenarioTesting(string cloudControllerResponse, HttpStatusCode steeltoeStatusCode, string? errorMessage,
```

Comment:
Please add an assertion on what's logged, similar to the first test [here](https://github.com/SteeltoeOSS/Steeltoe/blob/d306c94e067a4a6df4e3aa5135c84464e1518a76/src/Management/test/Endpoint.Test/Actuators/ThreadDump/EventPipeThreadDumperTest.cs#L26).